src/Entity/Configuration/LoanOfficerConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Entity.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Entity.Configuration
+{
+    public class LoanOfficerConfiguration : IEntityTypeConfiguration<LoanOfficer>
+    {
+        public void Configure(EntityTypeBuilder<LoanOfficer> builder)
+        {
+            builder.HasData(
+                new LoanOfficer
+                {
+                    Id = new Guid("9A07B75E-E943-48BC-821C-76685161668D"),
```

Comment:
You shouldn't use a verbatim Guid value. Instead, you should generate a random Guid as seen here https://stackoverflow.com/a/2344110/4815571

Reply:
consider
```suggestion
                    Id = Guid.NewGuid(),
```